src/App.jsx

```diff
@@ -1,33 +1,23 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Routes, Route } from 'react-router-dom'
 import './App.css'
+import Header from '../Components/Header'
+import ArticlesList from '../Components/ArticlesList';
+import DropdownBar from '../Components/DropdownBar';
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([])
```

Comment:
If you're not using this in here, then you don't need it here. This could be moved to "ArticlesList". State should be as low down as possible and as high up as needed.